.travis.yml

```diff
@@ -48,16 +48,26 @@ matrix:
   allow_failures:
     - rust: nightly
   include:
-    - rust: stable
+    - name: Backend
+      rust: stable
       before_install:
-        - nvm install 10
-        - npm ci
         - rustup component add rustfmt-preview
         - rustup component add clippy-preview
       script:
         - cargo fmt -- --check
         - cargo clippy --all-targets --all-features --all
         - cargo test
+    - name: Frontend
+      rust: stable
```

Comment:
do we need `rust` here at all? 

```yaml
language: node_js
node_js: 10
```
 should automatically invoke `nvm install`

Reply:
Does it work. I thought that the `language:` wouldn't be applied at the Build Matrix level.

Reply:
Travis CI's documentation regarding using multiple languages also doesn't use `language:` on Build Matrix.

https://docs.travis-ci.com/user/customizing-the-build#installing-a-second-programming-language

Reply:
https://github.com/skylines-project/skylines/blob/43a68499581ea0ad04ee70e40a6f3a1f49683928/.travis.yml#L81-L82

seems to work though 😉